queries.sql

```diff
@@ -33,7 +33,7 @@ COMMIT;
 select * from animals;
 select count(*) from animals;
 
-select count(*) from animals where escape_attempts = 0;
+select count(*) from animals where escape_attempts = 0 ;
```

Comment:
- [ Required ] Please consider adding `SQL QUERY` statements in the `queries.sql` file that answer the following questions as specified in the project requirements. Please see the screenshot below for more details on this.

## Project requirements
![Screenshot from 2023-01-21 02-42-52](https://user-images.githubusercontent.com/16392046/213824349-89a48322-d3e7-4b5e-bffe-2741d1de8442.png)